draft-ietf-rtcweb-jsep.xml

```diff
@@ -1627,16 +1627,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         <section title="addIceCandidate" anchor="sec.addicecandidate">
 
           <t>The addIceCandidate method provides a remote candidate to
-          the ICE agent, which, if parsed successfully, will be added
+          the ICE agent which will be added
           to the current and/or pending remote description according to
           the rules defined for Trickle ICE. The pair of MID and ufrag
           is used to determine the m= section and ICE candidate
           generation to which the candidate belongs. If the MID is not
           present, the m= section index is used to look up the locally
           generated MID (see
           <xref target="sec.applying-a-remote-desc" />), which is used
-          in place of a supplied MID. If these values or the candidate
-          string are invalid, an error is generated.</t>
+          in place of a supplied MID.
+          If the remote candidate is not successfully parsed, or these
```

Comment:
How is "not successfully parseD" different from the candidate string being invalid